config/routes.rb

```diff
@@ -2,6 +2,12 @@
   devise_for :users
 
   root 'users#index'
+
+
+  get '/api/users/:user_id/posts', to: 'api/posts#index'
+  get '/api/comments/:post_id', to: 'api/comments#index'
+  get '/api/comments/:user_id/:post_id/:text', to: 'api/comments#create'
```

Comment:
- [ ] Since comments belong to a post and a post belongs to a user, the endpoint should be like `/api/users/:user_id/posts/:id/comments` should be the get request for this API. Line 9 should be a POST request rather than a GET. And should seem like this: `/users/:user_id/posts/:id/comments`. Kindly fix it.
![image](https://github.com/shayan1234554321/The-Knowledge-Hub/assets/91486791/1445cf78-b176-4b35-9bd1-6d774eb74e16)